### DISCUSSION TITLE
Monospace (and non-monospace) fonts are not working (even with use of information from FAQ page)

### DISCUSSION THREAD

**Discussion by goobleplax** [kovidgoyal/kitty#4143] [Q&A]
I installed kitty assuming that it had great font support.

It's okay so far, but now that I'm installing fonts, nothing is working.

An example font is King Plus Monospace: https://quikscript.neocities.org/qprojects/qfonts.htm

The FAQ (https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font) states that monospaced fonts without the 'spacing=100' can still be used by including a config file.

Even after doing that, it's not working.

And when I added a request for support in the issues page, I was redirected here, and the project creator (kovidgoyal) complained that I was commenting "passive aggressive shit".

What...? No. It's aggravating that a terminal emulator can't display fonts correctly. Several other people have mentioned essentially the exact same problem and not received useful answers.

Kitty could be a great terminal emulator. The key to a terminal is display of text. The FAQ states that even _imperfect_ fonts can be displayed using a config file to override the default 'spacing=100' setting. And there are no other alterations to the configuration since it's a new install.

So if anyone has had similar experience and overcome it, or has any new ideas about perhaps making Kitty more capable of rendering a variety of fonts correctly, that would be great. Kitty could be great software if there was enough effort to make it work for the majority of fonts that people might use beyond a handful that are "perfect"; otherwise, Kitty is handicapped unnecessarily and it's unfortunate. So hopefully someone has insight and useful thoughts?

**Comment by goobleplax** [comment]
In case anyone finds this topic: the answer is [alacritty](https://alacritty.org/).

**Comment by Strykar** [comment]
You are either trolling or just combative, that link lists fonts from what 2003?
Have you tried a modern font from say, 20 years ahead?

FYI, I found Kitty after being unable to view a font correctly in Alacritty.

The solution for that problem was to add `adjust_line_height 5` to kitty.conf, an option which does not exist on Alacritty.
